src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
```

Comment:
Not clear naming.  Although there are no restrictions about using digitals in variable names in Java, in this case, "1" is similar to "l", so I'm suggesting using smth like this
```suggestion
        Lottery ball_1 = new Lottery();
```